tests/scripts/findkernelversion.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [[ "${SKIP_INSTALL}" == "true" ]]; then
+    echo "Skipping install: SKIP_INSTALL=${SKIP_INSTALL}"
+    exit 0
+fi
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "${SCRIPT_DIR}"/.definitions.sh
+
+export REGCTL_VERSION=v0.4.7
+mkdir -p bin
+curl -sSLo bin/regctl https://github.com/regclient/regclient/releases/download/${REGCTL_VERSION}/regctl-linux-amd64
+chmod a+x bin/regctl
+export PATH=$(pwd)/bin:${PATH}
+DRIVER_BRANCH=$(echo "${TARGET_DRIVER_VERSION}" | cut -d '.' -f 1)
+KERNEL_FLAVOR=$(uname -r | awk -F'-' '{print $3}')
```

Comment:
Since the AMI we are using comes with the `aws` kernel by default, this means the pipeline will only test the `aws` kernel flavor. In order to test all the kernel flavors, I am thinking we could pass the `KERNEL_FLAVOR` as input and have a matrix of test jobs -- one per kernel flavor.

Reply:
kernel flavor should come from server :   we can install any flavor(generic, nvidia) on target